src/tests/functional/plugin/gpu/shared_tests_instances/low_precision_transformations/fake_quantize_transformation.cpp

```diff
@@ -43,10 +43,11 @@ const std::vector<FakeQuantizeTransformationParam> fakeQuantizeOnDataValues = {
         { 256ul, {}, { 0.f }, { 2.55f }, { -1.28f }, { 1.27f } },
         "Pooling", "I8"
     },
-    {
-        { 256ul, {}, { 0.f }, { 2.55f }, { 2.55f }, { 2.55f } },
-        "Pooling", "U8"
-    },
+    // nGraph: dot interval FQ node is being const-folded
+    //{
+    //    { 256ul, {}, { 0.f }, { 2.55f }, { 2.55f }, { 2.55f } },
+    //    "Pooling", "U8"
+    //},
     {
```

Comment:
This test cannot success after FQ is being const-folded, because GPU plugin will also const-fold Pooling layer following FQ node, thus Pooling layer is gone.

Reply:
Can we leave this test-case but don't check Pooling precision?

Reply:
OK